perun/logic/commands.py

```diff
@@ -11,8 +11,6 @@
 
 from operator import itemgetter
 
-import colorama
```

Comment:
I removed colorama during the dependencies pruning as it felt a bit unnecessary in the current state of Perun. Since it is currently not possible to deploy or test Perun on Windows machines (and it will likely require a significant amount of work to do so), having a package that is aimed at fixing windows terminal output seems like needless dependency. Especially so since we are not able to test if it works as intended or not.

Reply:
Ook, fine decision.